docs/Deployment/BackupRestore.md

```diff
@@ -178,33 +178,41 @@ Creating letsencrypt-nginx-proxy-companion     ... done
 
 ```
 
-## How does a BTCPay Server Restore work? 
+## How does the BTCpay-restore.sh work? 
 
-The starting procedure is similar to the `btcpay-backup.sh`. 
-But instead of running backup, we run the `btcpay-restore.sh` script. 
-This will start the restore process.
+It's very similar to the `btcpay-backup.sh` process but in reverse. 
+The `btcpay-restore.sh` script will search for the `backup.tar.gz` file. 
+
+First off, open a terminal and type the following as root. Remember that if you set `BTCPAY_BACKUP_PASSPHRASE` on the backup, you have to set it on the restore too ; 
 
 ```
 cd "$BTCPAY_BASE_DIRECTORY"
-./btcpay-restore.sh
+./btcpay-restore.sh BTCPAY_BACKUP_PASSPHRASE 
 ```
-
-The script will start with the same credential check as the backup. 
-Again, this script needs to be run as `root`.
-
-Did you run the backup with `BTCPAY_BACKUP_PASSPHRASE` ? Then you do have to run the restore with this tag aswell. 
+This will start the restore process, it start by looking for the `backup.tar.gz` file.
+If it can't find the file in the right path, the script will give out an error.  
```

Comment:
```suggestion
This will start the restore process by unpacking the backup file.
If it can't find the file in the provided path, the script will give out an error.  
```